modules/hello-world/pages/start-using-sdk.adoc

```diff
@@ -39,7 +39,7 @@ after walking through a quick installation.
 
 == Before You Start
 
-Couchbase Capella, our Database-as-a-Service, lets you get on with what matters, while we take care of the administration for you.
+Couchbase Capella, the Database-as-a-Service, lets you get on with what matters, while it takes care of the administration for you.
```

Comment:
As we discussed last time, going away from the scope of the ticket is an inefficient way of tackling style inconsistencies which can occur across all 12 SDKs.
As discussed previously, this is something that needs tackling efficiently in a way that improves all of the SDK docs.

Besides, the use of the definite article here does not feel strictly correct.